components/org.wso2.carbon.identity.api.idpmgt.endpoint/src/gen/java/org/wso2/carbon/identity/api/idpmgt/endpoint/dto/AuthenticationDTO.java

```diff
@@ -0,0 +1,84 @@
+package org.wso2.carbon.identity.api.idpmgt.endpoint.dto;
+
+import io.swagger.annotations.ApiModel;
+import java.util.ArrayList;
+import java.util.List;
+import org.wso2.carbon.identity.api.idpmgt.endpoint.dto.AuthenticatorDTO;
+import org.wso2.carbon.identity.api.idpmgt.endpoint.dto.JITProvisioningDTO;
+import org.wso2.carbon.identity.api.idpmgt.endpoint.dto.PropertyDTO;
+
+import io.swagger.annotations.*;
+import com.fasterxml.jackson.annotation.*;
+
+import javax.validation.constraints.NotNull;
+
+
+
+/**
+ * Authentication related configurations
+ **/
+
+
+@ApiModel(description = "Authentication related configurations")
+public class AuthenticationDTO  {
+  
+  
+  
+  private JITProvisioningDTO jitProvisioning = null;
+  
+  
+  private List<AuthenticatorDTO> authenticators = new ArrayList<AuthenticatorDTO>();
+  
+  
+  private List<PropertyDTO> properties = new ArrayList<PropertyDTO>();
+
+  
+  /**
+   **/
+  @ApiModelProperty(value = "")
+  @JsonProperty("jitProvisioning")
+  public JITProvisioningDTO getJitProvisioning() {
+    return jitProvisioning;
+  }
+  public void setJitProvisioning(JITProvisioningDTO jitProvisioning) {
+    this.jitProvisioning = jitProvisioning;
+  }
+
+  
+  /**
+   **/
```

Comment:
Auto generated fragment for method comments. Shall we complete or . remove if it is a simple get/set method.